tokio/src/lib.rs

```diff
@@ -384,7 +384,14 @@
 //! [unstable features]: https://internals.rust-lang.org/t/feature-request-unstable-opt-in-non-transitive-crate-features/16193#why-not-a-crate-feature-2
 //! [feature flags]: https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section
 //!
-//! ## WASM support
+//! ## Supported platforms
+//!
+//! Tokio supports any platform that mio supports. You can find the list [in
+//! mio's documentation][mio-supported].
```

Comment:
I would kinda favor listing some of the most important supported platforms here as well, which I think are very unlikely to go away anytime soon. For example:

* Android (API level 21)
* Linux
* Windows
* iOS
* macOS
* FreeBSD

I would be happy to open a separate PR to contribute this proposal.

Reply:
That's a good idea. I'll add that list here.